rero_ils/modules/loans/utils.py

```diff
@@ -41,11 +41,9 @@ def get_circ_policy(loan, checkout_location=False):
     :return the cipo related to the loan
     """
     item = Item.get_record_by_pid(loan.item_pid)
-    library_pid = None
-    if checkout_location:
-        library_pid = loan.checkout_library_pid
-    if not library_pid:
-        library_pid = loan.library_pid
```

Comment:
We have one more condition in the original code. Even if we have a `checkout_location` but the `loan.checkout_library_pid` we will use the ` loan.library_pid` !